lib/apiservers/engine/backends/container/viccontainer.go

```diff
@@ -15,21 +15,15 @@
 package container
 
 import (
-	"fmt"
-	"strings"
-
 	containertypes "github.com/docker/engine-api/types/container"
 )
 
-// DefaultEnvPath defines the default PATH environment variable to use in a container
-const DefaultEnvPath = "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
-
 // VicContainer is VIC's abridged version of Docker's container object.
 type VicContainer struct {
 	Name        string
-	ID          string
+	ID          string // Image ID the container was created from
```

Comment:
Minor, but it might be worth just renaming this to `ImageID` .. I seem to remember it causing some confusion previously. No biggie otherwise, this comment definitely helps make it clear.